features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimelineItemEventForTimestampViewProvider.kt

```diff
@@ -33,5 +34,9 @@ class TimelineItemEventForTimestampViewProvider : PreviewParameterProvider<Timel
             aTimelineItemEvent(
                 messageShield = MessageShield.UnknownDevice(isCritical = true),
             ),
+            aTimelineItemEvent(
+                content = aTimelineItemRedactedContent(),
+                messageShield = MessageShield.SentInClear(isCritical = true),
+            ),
```

Comment:
Let's remove the new preview, I will add it again later.

```suggestion
            // aTimelineItemEvent(
            //    content = aTimelineItemRedactedContent(),
            //    messageShield = MessageShield.SentInClear(isCritical = true),
            //),
```